phpunit.xml.dist

```diff
@@ -17,6 +17,10 @@
         </testsuite>
     </testsuites>
 
+    <listeners>
```

Comment:
Couldn't we manage this file through dev-it?

Reply:
That would be a good idea I think. Now that we have a standard structure.